store/modules/generator/generator.actions.ts

```diff
@@ -298,49 +278,77 @@ export const actions: Actions<State, RootState> = {
   ): Promise<void> {
     //fetch screenshots from each URL
     // receive screenshots and add src to manifest
-    console.log(urls);
-    console.log();
-    await fetch(screenshotsUrl + '/screenshotsAsBase64Strings', {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json',
-      },
-      body: JSON.stringify({
-        url: urls,
-      }),
-    }).then((res) => {
-      res.json().then((jsonData) => {
-        console.log(jsonData.images);
-        this.$axios
-          .$post(
-            `${apiUrl}/${state.manifestId}/generateMissingScreenshots`,
-            jsonData.images
-          )
-          .then((result) => {
-            console.log('Result');
-            console.log(result);
-            commit(types.OVERWRITE_MANIFEST, result);
-            commit(types.ADD_ASSETS, result.assets);
-            dispatch('update');
-          });
+
+    try {
+      const res = await fetch(screenshotsUrl + "/screenshotsAsBase64Strings", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          url: urls,
+        }),
       });
-    });
+
+      if (res.ok) {
+        interface ScreenshotsAsBase64StringsInterface {
+          images: Array<Icon>;
+        }
+
+        const response: ScreenshotsAsBase64StringsInterface = await res.json();
+        console.log(response);
+
+        console.log("Result", res, response, state);
+        response.images.forEach((img) => {
+          img.src = "data:" + (img.type || "image/png") + ";base64," + img.src;
+        });
+
+        commit(types.ADD_SCREENSHOT, response.images);
+        commit(types.UPDATE_MANIFEST_PARTIAL, {
+          screenshots: state.screenshots,
+        });
+        dispatch("update");
+      } else {
+        throw "screenshot service error";
+      }
+    } catch (e) {
+      console.error(e);
+    }
   },
   async generateMissingImages(
     { commit, state, dispatch },
     iconFile: File
   ): Promise<void> {
-    let formData = new FormData();
-    formData.append('file', iconFile);
+    let result: Array<Icon> = [];
+
+    try {
+      const form = new FormData();
+      form.append("baseImage", iconFile);
+      form.append("padding", "0");
+      //form.append('color', '#0000000');
+      form.append("colorChanged", "false");
+      form.append("platform", "windows10");
+      form.append("platform", "android");
+      form.append("platform", "ios");
+
```

Comment:
using the imageGeneratorUrl that matches the new service:
https://github.com/pwa-builder/PWABuilder/blob/v3.0/src/script/services/icon_generator.ts#L29